Header header

float32 brake_cmd #Nm
float32 brake_output #Nm
bool enabled

#Not to worry about those
#brake light
bool boo_input
bool boo_cmd
bool boo_output

#status
bool override #driver override
bool driver #drive activity
bool timeout #command timeout

